R/utils_render_rtf.R

```diff
@@ -1244,18 +1244,17 @@ create_body_component_rtf <- function(data) {
     if (stub_available && summaries_present &&
         i %in% groups_rows_df$row_end) {
 
-      group <-
-        groups_rows_df %>%
-        dplyr::filter(row_end == i) %>%
-        dplyr::pull(group)
+      group_id <-
+        groups_rows_df[
+          groups_rows_df$row_end == i &
+            !is.na(groups_rows_df$row_end), ][["group_id"]]
```

Comment:
```r
groups_rows_df[na.omit(groups_rows_df$row_end == i), "group_id", drop = TRUE]
```

Reply:
Replaced statement with your suggestion. Thank you!